packages/frontend/src/pages/UserActivity.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { Callout, Card, Colors, H5, Tabs } from '@blueprintjs/core';
+import { Tooltip2 } from '@blueprintjs/popover2';
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+    display: grid;
```

Comment:
@IrakliJani what do you think of this `grid display` I generated with https://grid.layoutit.com/ , it was easier to add than using flex. You can test it here https://github.com/lightdash/lightdash/pull/4235/files#diff-712217320621825cce9c84413c1e4815975b5d316c42ec3c1c87505c8df7f5d1R5

![image](https://user-images.githubusercontent.com/1983672/214342673-8478a3c0-cd2a-4aaa-9258-0f96dd38ef34.png)